tsconfig.base.json

```diff
@@ -42,6 +42,9 @@
       "@island.is/api-catalogue/types": [
         "libs/api-catalogue/types/src/index.ts"
       ],
+      "@island.is/api-domains-sessions": [
```

Comment:
Slashes? :/

Reply:
Yup guess the new nx convention got the better of me here.